libraries/shared/watcher/storage_watcher_test.go

```diff
@@ -86,6 +89,17 @@ type ExecuteInput struct {
 	statusWriter *fakes.MockStatusWriter
 }
 
+type MockThrottler struct {
```

Comment:
nbd but I think conventionally we've been putting these kinda objects in their own files at `libraries/shared/mocks`

Reply:
👍